File: repos/beecrowd/python/bee1099.py
n = int(input())
for i in range(n):
    soma = 0
    n1, n2 = map(int, input().split())
    if n1 < n2:
        for j in range(n1+1, n2):
            if j % 2 != 0:
                soma += j
        print(soma)
    else:
        for j in range(n2+1, n1):
            if j % 2 != 0:
                soma += j
        print(soma)

File: javascript/bee1099.js
const input = require("fs").readFileSync("stdin", "utf8").split("\n");
const [n, ...lines] = input;

lines.forEach((line) => {
    const [n1, n2] = line.trim().split(" ").map(Number);
    let soma = 0;

    const menor = Math.min(n1, n2); 
	const maior = Math.max(n1, n2);

	for (let i = menor + 1; i < maior; i++) {
		if (i % 2 !== 0) {
			soma += i;
		}
	}
    console.log(soma);
})

